building/build_options.adoc

```diff
@@ -0,0 +1,67 @@
+# Options for building JBoss Tools
+
+## Maven options
+
+Here are the options we use for JBoss Tools builds:
```

Comment:
the options below is a list of flags that we don't use together in CI builds - its more a duplication of mvn docs.

Maybe it was meant to say "the options we recommend to learn/use with JBoss Tools builds" ?